activities/staff-management/expense.md

```diff
@@ -40,3 +40,7 @@ After approval the reimbursement is added to your next salary payment.
 ## How to request reimbursement for contractors and volunteers
 
 Contractors and volunteers should complete the expense form provided by their contracting contact.
+
+## Expense forms
+
+Staff uses Tentoo NMBRS to have expenses reimbursed. We send contractors and other externals [this form.](https://docs.google.com/document/d/1zt7roOx9YOwH7fnvmeA4cS7fCmvQGUWHBtx2knJ4Ml4/edit)
```

Comment:
When I follow this link I get a really weird document

Reply:
@demkodo is this the right document?

Reply:
It wasn't! Apologies for the delayed response - solved it